### PR TITLE
Ram 1500

### PR DESCRIPTION
**Pull request by adeebshihadeh**

### CODE REVIEW

**Comment by jyoung8607** [comment]
This work supersedes `stellantis_dasm.dbc`; it can be garbage-collected in this PR.

File: chrysler_ram_1500.dbc [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+    SG_MUL_VAL_
+
+BS_:
+
+BU_: XXX
+
+
+BO_ 35 STEERING: 8 XXX
+ SG_ STEER_ANGLE : 4|13@0+ (0.3187251,-1307.888) [-360|360] "deg" XXX
+ SG_ STEERING_RATE : 20|13@0+ (0.3187251,-1305.498) [0|8191] "deg/s" XXX
```
Comment (sshane) [inline chrysler_ram_1500.dbc:41]: Looks like you accidentally copied the old signal definitions too, openpilot is learning ~9 for the steering ratio when it should be 16.7.

```suggestion
 SG_ STEER_ANGLE : 5|14@0+ (0.5,-2048) [-2048|2047] "deg" XXX
 SG_ STEERING_RATE : 21|14@0+ (0.5,-2048) [-2048|2047] "deg/s" XXX
```

File: chrysler_ram_dt_generated.dbc [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+BO_ 250 DAS_6: 8 XXX
+ SG_ LKAS_ICON_COLOR : 1|2@0+ (1,0) [0|3] "" XXX
+ SG_ LKAS_LANE_LINES : 19|4@0+ (1,0) [0|1] "" XXX
+ SG_ LKAS_ALERTS : 27|4@0+ (1,0) [0|1] "" XXX
+ SG_ CAR_MODEL : 15|8@0+ (1,0) [0|255] "" XXX
+
+BO_ 720 BSM_1: 6 XXX
+ SG_ LEFT_STATUS : 0|3@1+ (1,0) [0|0] "" XXX
+ SG_ RIGHT_STATUS : 3|3@1+ (1,0) [0|0] "" XXX
```
Comment (adeebshihadeh) [inline chrysler_ram_dt_generated.dbc:84]: @realfast this is quite different than the old definition. are the definitions the same for the existing Chrysler platform and Ram DT? what are these extra two bits?
```
BO_ 720 BLIND_SPOT_WARNINGS: 6 XXX
 SG_ BLIND_SPOT_RIGHT : 5|1@0+ (1,0) [0|1] "" XXX
 SG_ BLIND_SPOT_LEFT : 2|1@0+ (1,0) [0|1] "" XXX
```

Comment (adeebshihadeh) [reply chrysler_ram_dt_generated.dbc]: I reverted to the old one for both Ram and Chrysler. Let's move to the new one in a separate PR.

**Pull request merged**